addons/binding/org.openhab.binding.netatmo/ESH-INF/thing/welcomeCamera.xml

```diff
@@ -0,0 +1,113 @@
+            <channel id="welcomeCameraVpnUrl" typeId="vpn_url"></channel>
+            <channel id="welcomeCameraIsLocal" typeId="is_locale"></channel>
+                                    
+        </channels>
+              
+        <properties>
+             <property name="actualApp">app_welcomecamera</property>
+        </properties>         
+        
+        <config-description>
+            <parameter name="welcomeHomeId" type="text" required="true" readOnly="true">
+            	<label>Home ID</label>
+            	<description>Id of the home</description>
+            </parameter>
+
+            <parameter name="welcomeCameraId" type="text" required="true" readOnly="true">
+            	<label>Person ID</label>
+            	<description>Id of the camera</description>
+            </parameter>
+
+        </config-description>      
+	                          
+    </thing-type>
+
+                  
+    <channel-type id="id">
+        <item-type>String</item-type>
+        <label>Id</label>
+        <description>ID of the camera</description>
+        <state readOnly="true">
+        </state>
+    </channel-type>
+    <channel-type id="type">
+        <item-type>String</item-type>
+        <label>Type</label>
+        <description>Type of the camera</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    <channel-type id="status">
+        <item-type>String</item-type>
+        <label>State</label>
+        <description>State of the camera</description>
+        <state readOnly="true">
+            <options>
+                <option value="on">on</option>
+                <option value="off">off</option>
+            </options>
+        </state>       
+    </channel-type>
+    <channel-type id="sd_status">
+        <item-type>String</item-type>
+        <label>SD State</label>
+        <description>State of the SD card</description>
+        <state readOnly="true">
+            <options>
+                <option value="on">on</option>
+                <option value="off">off</option>
+            </options>
+        </state>   
+    </channel-type>
+    <channel-type id="alim_status">
+        <item-type>String</item-type>
+        <label>Alim State</label>
+        <description>State of the power connector</description>
+        <state readOnly="true">
+            <options>
+                <option value="on">on</option>
+                <option value="off">off</option>
+            </options>
+        </state>   
+    </channel-type>
+    <channel-type id="name">
+        <item-type>String</item-type>
+        <label>Camera name</label>
+        <description>Name of the camera</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    <channel-type id="vpn_url">
+        <item-type>String</item-type>
+        <label>VPN Url</label>
+        <description>Only for scope access_camera. Address of the camera</description>
```

Comment:
Havent implemented the images till now, because problems i have problems with the response of the swagger api. The response type file or byte didn't work in the swagger api just String, so i can't save the image object or have not idea howto.
The vpn_url is the response of the url described on the netatmo site, i just save the url as i get it from the netatmo api, maybe i also could handle the operations described on the netatmo site. Image and video sitemap items are new to me ill give it a try, how to implement them or is this just in the PR #2412 you sent?